ui/src/components/LoginScreen.tsx

```diff
@@ -57,11 +83,25 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
     <Tile key='jwt'><JWTLoginForm onLogin={onLogin}/></Tile>
   ];
 
-  const tiles = deploymentMode === DeploymentMode.PROD_DABL ? dablTiles : localTiles;
+  const setupRequiredTile = (
+    <Tile key='test' header={logoHeader}>
+      <div className='setup-required'>
+        <h3>Welcome to the Daml Open Marketplace!</h3>
+        <p>
+          It looks like you have not completed the necessary deployment steps to configure this app.
+          Please create an Operator role contract for the UserAdmin party, and deploy the necessary triggers.</p>
+        <h4>See <a href='https://github.com/digital-asset/da-marketplace#add-the-operator-role-contract'>here</a> for more information.</h4>
```

Comment:
btw in the screenshot from the other day, the default blue/purple link color looked pretty hard to read against the dark background.

Maybe add the className `"dark"` to the `a`, and have the color of `a.dark` tags be [var(--blue-80)](https://www.figma.com/file/HxPETLG8mqxF00IIod5gY6/Daml-Open-Marketplace-WF?node-id=1080%3A1482) somewhere in css

Reply:
that looks much better!